src/tools/rbd/action/Snap.cc

```diff
@@ -29,13 +43,23 @@ int do_list_snaps(librbd::Image& image, Formatter *f)
   if (r < 0)
     return r;
 
+  if (!all_snaps) {
+    snaps.erase(remove_if(snaps.begin(),
+			  snaps.end(),
+			  boost::bind(is_not_user_snap_namespace, &image, _1)),
+		snaps.end());
```

Comment:
This indentation is meant to show that snaps.end() is erase's argument as well as remove_id. Do you want me to join it all in one line and wrap based on line length?